common4j/src/main/com/microsoft/identity/common/java/exception/ArgumentException.java

```diff
@@ -32,6 +30,10 @@
     public final static String ACQUIRE_TOKEN_OPERATION_NAME = "acquireToken";
     public final static String ACQUIRE_TOKEN_SILENT_OPERATION_NAME = "acquireTokenSilent";
     public final static String BROKER_TOKEN_REQUEST_OPERATION_NAME = "brokerTokenRequest";
+    public final static String GET_ACCOUNTS_OPERATION_NAME = "getAllAccounts";
```

Comment:
Can we do something else, so that we're not curating a separate list of these?

Reply:
I think this is fine for now.